src/link-builder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Link_Builder interface.
+ *
+ * @package Duplicate_Post
+ */
+
+namespace Yoast\WP\Duplicate_Post;
+
+interface Link_Builder {
+
+	/**
+	 *  Build URL for duplication action.
```

Comment:
Method descriptions should start with a third-person verb.
Also: there's a double space at the beginning.

Note: not my area of expertise thus I'm not sure using an interface is worth it.

Reply:
Will fix the CS.

It's not needed at the moment but I anticipated that it could be when the other two links are ported to the new code from the legacy one.